appveyor.yml

```diff
@@ -21,13 +23,12 @@ cache:
 init:
   - ps: $Env:sdkbin = "C:\Program Files\Microsoft SDKs\Windows\" + $Env:sdkver + "\Bin"
   - ps: $Env:sdkverpath = "C:/Program Files/Microsoft SDKs/Windows/" + $Env:sdkver + "/Setup/WindowsSdkVer.exe"
-  - ps: $Env:path = $Env:python + ";" + $Env:python + "\scripts;" + $Env:sdkbin + ";" + $Env:path
+  - ps: $Env:path = "C:\Enthought\edm;" + $Env:sdkbin + ";" + $Env:path
 
 install:
-  - ps: if ((Test-Path "c:/temp") -eq 0) { mkdir c:/temp }
-  - ps: python -m pip install --upgrade --no-binary wheel pip
-  - ps: pip install --upgrade wheel
-  - ps: pip --version
-  - cmd /v:on /e:on /c ".\appveyor-install.cmd"
+  - cmd: appveyor-bootstrap-edm.cmd setup
+  - ps: edm info
+  - cmd /e:on /v:on /c ".\appveyor-install.cmd"
```

Comment:
there is probably no need for a separate install and test batch files. We can probably call edm run from appveyor.yml directly